### PR TITLE
chore: Just check dependencies' license with apache action

### PR DESCRIPTION
**Pull request by JackyYangPassion**

### CODE REVIEW

**Review by zyxxoo — DISMISSED** [review]

**Review by javeme — DISMISSED** [review]

**Review by imbajin — CHANGES_REQUESTED** [review]
we should handle or know how to fix the problem before merge it

image

**Comment by imbajin** [comment]
and also fix dependency-review by github action

image

File: .github/workflows/licence-checker.yml [hunk 1]
```diff
@@ -22,6 +22,27 @@ jobs:
           token: ${{ github.token }}
           config: .licenserc.yaml
 
+  dependency-license:
```
Comment (javeme) [inline .github/workflows/licence-checker.yml:25]: can we rename to `check-dependency-license` and also rename `check-license` to `check-source-license`

**Comment by javeme** [comment]
Is it still in progress?

**Comment by JackyYangPassion** [comment]
Close the PR first and then submit it if there is progress later.

**Pull request closed without merging**